units/commit_queue.go

```diff
@@ -563,9 +563,13 @@ func AddMergeTaskAndVariant(patchDoc *patch.Patch, project *model.Project, proje
 	}
 
 	mergeTask := model.ProjectTask{
-		Name: evergreen.MergeTaskName,
-		Commands: []model.PluginCommandConf{
-			{
+		Name:      evergreen.MergeTaskName,
+		DependsOn: dependencies,
```

Comment:
can we move this logic to a helper function that returns `[]PluginCommandConf` so we can instantiate the mergeTask in one block? Something like:

```
mergeTask := model.ProjectTask{
		Name:      evergreen.MergeTaskName,
		DependsOn: dependencies,
                 Commands: getMergeTaskCommands(source)
             }
```

Reply:
Done.